HISTORY.md

```diff
@@ -1,5 +1,13 @@
 # History
 
+## [0.2.0-rc2] (2023-12-11)
```

Comment:
Isn't the mars Request support for multiple params also in that release?

Reply:
line 8?